app/src/redux/modules/groups/actions/create.js

```diff
@@ -40,6 +40,8 @@ export default function createGroup({ name, members }) {
     return axios
       .post(url, body)
       .then(result => dispatch(createGroupSuccess(result.data)))
-      .catch(error => dispatch(createGroupFailure(error)));
+      .catch(error => {
```

Comment:
Is there a reason why you added those brackets? They are not needed

Reply:
I had a console log in there to spit out the error for a debugging thing. I forgot to remove it when I removed the log.